### PR TITLE
support aurora instance's parameter group and modifyinstance

### PR DESCRIPTION
**Pull request by arcadiatea**
This change for #6841.

### CODE REVIEW

**Comment by catsby** [comment]
Hey @arcadiatea – am I correct that we can simply use the existing `aws_db_parameter_group`, correct?

File: builtin/providers/aws/resource_aws_rds_cluster_instance.go [hunk 1]
```diff
@@ -58,13 +58,31 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
...
-				ForceNew: true,
+			},
+
+			"db_parameter_group_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"status": &schema.Schema{
```
Comment (catsby) [inline builtin/providers/aws/resource_aws_rds_cluster_instance.go]: This attribute doesn't seem to be set anywhere, what is it meant to record? The `DBInstanceStatus`, or `status` from `StatusInfos.DBInstanceStatusInfo.N` ?

- http://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBInstance.html

**Comment by catsby** [comment]
I have a question about the `status` attribute but I did test out this branch and it worked as expected 👍

**Comment by arcadiatea** [comment]
Hey, @catsby
This change is to allow Aurora instance be able to use customized parameter group set by aws_db_parameter_group.
The current issues is Aurora instance always created with default db parameter group and not be able to detect the modify requirement.

**Comment by catsby** [comment]
@arcadiatea yep! I tried it out and it works as expected. I do have an question about the added `status` parameter; it's not being set, and it's not clear to me what value this should be. Do you recall?

**Comment by arcadiatea** [comment]
@catsby  that status attribute is not be used for this modification , you are right.
I am working on another enhancement feature on aws_rds_cluster_instance (support security_group changes)  and I can include the usage of status there.

**Comment by catsby** [comment]
Ok, thanks for clarifying! If you remove that attribute from this commit, I'll go ahead and merge this

**Comment by arcadiatea** [comment]
@catsby status attributes is removed. This change probably also fix #5710, #5436

**Comment by catsby** [comment]
Thanks! I'm going to merge this with an addition to the test file as well.

**Pull request merged**